packages/wallets/src/features/cfd/screens/ChangePassword/ContentTemplate.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { WalletText } from '../../../../components/Base';
+
+interface ContentTemplateProps {
+    bodyText: React.ReactNode;
+    button: React.ReactNode;
+    headingText: string;
+    icon: React.ReactNode;
+}
+
+const ContentTemplate: React.FC<ContentTemplateProps> = ({ bodyText, button, headingText, icon }) => (
```

Comment:
Dont be needing this. Just wait till this is merged https://github.com/binary-com/deriv-app/pull/11483